### DISCUSSION TITLE
GitHub Universe Scholarship 2023

### DISCUSSION THREAD

**Discussion by Debaditya-Som** [community/community#64571] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Is there any provision of scholarship for GitHub Universe 2023?

**Comment by mdxabu** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> Is there any provision of scholarship for GitHub Universe 2023?

yeah @Debaditya-Som ,They have an scholarship for github universe 2023
but now it is closed .
[scholarship link](https://github.blog/2019-08-12-apply-for-a-github-inclusivity-scholarship-and-join-us-at-github-universe/)

**Reply by Debaditya-Som** [reply, marked as answer]
But thts for 2019
Hopefully there's gonna be a link for 2023

**Comment by mdxabu** [comment]
> But thts for 2019 Hopefully there's gonna be a link for 2023
sorry ,maybe they'll open later
